data_steward/cdr_cleaner/cleaning_rules/create_person_ext_table.py

```diff
@@ -9,7 +9,6 @@
 deid to be correctly de-identified.
 This cleaning rule will populate the person_ext table
 The following fields will need to be copied from the observation table:
-src_id (from observation_ext, should all be “PPI/PM”)
 state_of_residence_concept_id: the value_source_concept_id field in the OBSERVATION table row where
```

Comment:
This same comment is present in the integration and unit test. I would also remove it from there.